FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -2330,10 +2334,10 @@ - (nullable FIRUser *)getStoredUserForAccessGroup:(NSString *_Nullable)accessGro
   } else {
 #if TARGET_OS_TV
     if (self.shareAuthStateAcrossDevices) {
-      FIRLogError(kFIRLoggerAuth, @"I-AUT000001",
-                  @"Getting a stored user for a given access group is not supported "
-                  @"on tvOS when `shareAuthStateAcrossDevices` is set to `true` (#8878)."
-                  @"This case will return `nil`.");
+      FIRLogWarning(kFIRLoggerAuth, @"I-AUT000018",
```

Comment:
Ah, I think this change was meant for line 653 of `FIRAuthBackend.m`.

Reply:
got it thx